osu.Game/Graphics/UserInterface/ExternalLinkButton.cs

```diff
@@ -42,6 +49,31 @@ public ExternalLinkButton(string link = null)
             };
         }
 
+        private void copyUrl()
+        {
+            host.GetClipboard()?.SetText(Link);
+            notificationOverlay.Post(new SimpleNotification
+            {
+                Text = "Copied URL!"
+            });
+        }
+
+        public MenuItem[] ContextMenuItems
+        {
+            get
+            {
+                List<MenuItem> items = new List<MenuItem>();
+
+                if (Link != null)
+                {
+                    items.Add(new OsuMenuItem("Open", MenuItemType.Standard, () => host.OpenUrlExternally(Link)));
+                    items.Add(new OsuMenuItem("Copy URL", MenuItemType.Standard, () => copyUrl()));
```

Comment:
As per the inspection:
```suggestion
                    items.Add(new OsuMenuItem("Copy URL", MenuItemType.Standard, copyUrl));
```